domain/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
```

```suggestion
    id("kotlin")
```

도메인 모듈은 순수 코틀린 모듈로 만들어보세요!